legacy/coverage/coverage.js

```diff
@@ -10,17 +10,17 @@ async function collectCoverage() {
     // search for reports
     const coverageFolder = __dirname;
     const report = {};
-    const getWorstCaseReport = (category) => {
+    const getMergedReport = (category) => {
         const reports = readdirSync(coverageFolder).filter(f => f.startsWith(`report-${category}`) && f.endsWith(".json")).map(f => require(join(coverageFolder, f)));
         const result = {};
         for (const feature of [].concat.apply([], reports.map(r => Object.keys(r)))) {
-            result[feature] = Math.min(...reports.map(r => r[feature] || 0));
+            result[feature] = Math.max(...reports.map(r => r[feature] || 0));

```

Comment:
So the logic here is that for every run, only one of the reports actually has coverage info, so we're looking for the max value of all reports since the rest will all be `0`?

Reply:
Correct.